Import/BibImportTest.php

```
@@ -0,0 +1,252 @@
+    }
+
+    public function testLookupReturnsNullWhenNotFound(): void
+    {
+        $this->assertNull(\pl_bibimport_lookup('9999'));
+    }
+
+    // --- pl_bibimport_is_duplicate -----------------------------------------
+
+    public function testIsDuplicateTrueWhenEntryExists(): void
+    {
+        \FakeDb::on('/FROM Entries/', [['EnId' => 1]]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\FakeDb' in method 'testIsDuplicateTrueWhenEntryExists'. (undefined)

(StaticAccess)